Packs/AutoFocus/ReleaseNotes/1_3_7.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Palo Alto Networks AutoFocus v2
+- Added support for URLs with non-ASCII characters in the ***url*** command.
+- Added the parameter **handle_error** to have an option to handle errors with 404, 409 codes.
```

Comment:
please use the display name, as `handle_error` is not visible/helpful to customers.